cpp/example_code/acm/tests/CMakeLists.txt

```diff
@@ -51,25 +51,25 @@ add_executable(
 )
 
 if (WINDOWS_BUILD)
-    if (AWSSDK_INSTALL_AS_SHARED_LIBS)
-        # set(BIN_SUB_DIR "/Debug") # If you are building from the command line, you may need to uncomment this
-        # and set the proper subdirectory to the executables' location.
+    # set(BIN_SUB_DIR "/Debug") # If you are building from the command line, you may need to uncomment this
```

Comment:
I see this in current code, so it's a question not a suggestion, but the comment makes me think it would be better as a flag?

Reply:
The idea is that someone tailors this to their particular environment. And I am trying to help them debug any issues.

This stuff seems to change with new major versions of build tools.